cmsis/device/rtos/mbed_lib.json

```diff
@@ -85,6 +85,12 @@
         "MCU_PSOC6_M4": {
             "target.macros_add": ["CY_RTOS_AWARE"]
         },
+        "ARM_MUSCA_B1": {
+            "mutex-num": 4,
+            "semaphore-num": 4,
+            "thread-num": 9,
```

Comment:
I simply copied them from ARM_MUSCA_S1, but are the values still valid?

Reply:
@gbrtth Would you be able to answer this one? Why these limits are set for MUSCA S1? I cant find the relevant information in the commit history.

Reply:
Hi, I have copied these values for Musca S1 from [here](https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/blob/678a381bb6fec55b45e256dd53bec7e94017e41d/mbed_app.json#L11). I don't know the reason why these values were selected, but I faced the same issue described above without configuring it.
I thought if it worked with the regression tests it would work with greentea too.

Reply:
@gbrtth Thanks for the reply. As per @Patater's comment below, we don't need to reserve most of the RTOS resources for general use. Only TF-M regression and PSA arch tests need more, and its enough to have the values overridden in mbed-os-tf-m-regression-tests. I've just force-pushed newer changes.